<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>270</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Janela Principal</string>
  </property>
  <widget class="QLineEdit" name="lineBuscaPlaca">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>AAA9999</string>
   </property>
   <property name="placeholderText">
    <string>Placa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcPlaca">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Procurar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCadCarro">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar Carro</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCarrosEstc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Carros estacionados:</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="twListagem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Placa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cidade</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Estado</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Modelo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Marca</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Entrada</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Listagem de Carros Cadastrados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaida">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar Sa√≠da</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
